apps/avifenc.c

```diff
@@ -132,7 +132,7 @@ static void syntaxLong(void)
     printf("    --crop CROPX,CROPY,CROPW,CROPH    : Add clap property (clean aperture), but calculated from a crop rectangle\n");
     printf("    --clap WN,WD,HN,HD,HON,HOD,VON,VOD: Add clap property (clean aperture). Width, Height, HOffset, VOffset (in num/denom pairs)\n");
     printf("    --irot ANGLE                      : Add irot property (rotation). [0-3], makes (90 * ANGLE) degree rotation anti-clockwise\n");
-    printf("    --imir MODE                       : Add imir property (mirroring). 0=top-to-bottom, 1=left-to-right\n");
+    printf("    --imir AXIS                       : Add imir property (mirroring). 0=top-to-bottom, 1=left-to-right\n");
```

Comment:
Here and at line 1446 I use the words "AXIS" and "axis" in the messages. I wonder if we should continue to use the word "MODE" and "mode" in the messages. Thoughts?

Reply:
The only reason we used "mode" anywhere was due to the renaming inside of the HEIF standard's draft, which ended up not being used. I think leveraging the original word "axis" in all of these places is appropriate.